codex/sales.nim

```diff
@@ -153,13 +153,19 @@ proc cleanUp(
   # Re-add items back into the queue to prevent small availabilities from
   # draining the queue. Seen items will be ordered last.
   if reprocessSlot and request =? data.request:
+    without collateral =?
+      await sales.context.market.slotCollateral(data.requestId, data.slotIndex):
```

Comment:
If you decide to raise from `slotCollateral`, be careful using `.catch` with `await` (swallows CancelledError)